com.snipaste.Snipaste.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.snipaste.Snipaste</id>
+  <name>Snipaste</name>
+  <project_license>LicenseRef-proprietary=https://www.snipaste.com/index.html</project_license>
+  <developer_name>Snipaste</developer_name>
+  <summary>Snipaste is not just a snipping tool. It will boost your productivity if you get used to its 'paste'!</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <launchable type="desktop-id">com.snipaste.Snipaste.desktop</launchable>
+  <url type="homepage">https://www.snipaste.com</url>
+  <url type="help">https://www.snipaste.com/support.html</url>
+  <description>
+    <p>Snipaste is not just a snipping tool. It will boost your productivity if you get used to its 'paste'!</p>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Snipaste.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://i.v2ex.co/71Ftp04b.png</image>
```

Comment:
if this is a temporary image host, better to add these in the PR or they will be lost.

Reply:
also why are these screenshots on windows?

Reply:
![image](https://github.com/user-attachments/assets/c2df18d0-e4b5-4ff6-b121-9b2fe8c99a3d)

Reply:
The official Snipaste has no other screenshots. https://www.snipaste.com/

Reply:
Try taking some screenshots on Linux and add them to the PR there. Then update the link to the metainfo.